src/main/java/com/microsoft/azure/springcloudplayground/generator/ProjectRequest.java

```diff
@@ -44,6 +44,13 @@
     @Setter
     private List<String> facets = new ArrayList<>();
 
+    @Getter
+    private Map<String, Integer> servicesPorts;
+
+    @Getter
+    @Setter
+    private Integer port;
```

Comment:
make it string?

Reply:
It there any reason we must use `String` for port ? The front-end input will ensure there will be valid integer or null. Or you need to handle this from back-end.